File: VersaoFinal/Latex/codigos/micro05.lua
local nome, sexo, x, h, m
h = 0
m = 0

for x = 1, 5, 1 do
    io.write("Digite o nome: ")
    nome = io.read()
    io.write("H - Homem ou M - Mulher: ")
    sexo = io.read()
    
    if sexo == 'H' then
      h = h + 1
    elseif sexo == 'M' then
      m = m + 1
    else
      io.write("Sexo só pode ser H ou M!\n")
    end
  
end

io.write("Foram inseridos ",h," Homens\n")
io.write("Foram inseridos ",m," Mulheres\n")

File: CC/Codigos_Fonte_Relatorio_1/micro05/micro05.pas
program micro05;

var
    nome, sexo : string;
    x, h, m : integer;    

begin
    h := 0;
    m := 0;

    for x:=0 to 4 do
    begin
        write('Digite o nome: ');
        read(nome);

        write('H - homem ou M - mulher: ');
        read(sexo);

        case sexo of
            'M' :
                m := m + 1;
            
            'H' :
                h := h + 1;    
        end;
    end;

    writeln('Foram introduzidos ',h,' homens.');
    writeln('Foram introduzidos ', m, ' mulheres.');

end.

